modules/friendlytag.js

```diff
@@ -1434,80 +1465,96 @@ Twinkle.tag.callback.evaluate = function friendlytagCallbackEvaluate(e) {
 		params.patrol = form.patrolPage.checked;
 	}
 
+	params.tags = form.getChecked( Twinkle.tag.mode + 'Tags' );
+
+	// Save values of input fields into params object. This works as quickform input
+	// fields within subgroups of elements with name 'articleTags' (say) have their
+	// name attribute as 'articleTags.' + name of the subgroup element
+
+	var name_prefix = Twinkle.tag.mode + 'Tags.';
+	$(form).find("[name^='" + name_prefix + "']").each(function(idx,el) {
```

Comment:
Wouldn't it be somewhat more parsimonious to add a negative selector for the checkboxes, since they're covered below?

```suggestion
	$(form).find("[name^='" + name_prefix + "']:not(input[type=checkbox])").each(function(idx,el) {
```

Reply:
To (slightly) improve selector efficiency, I think a better way is to use a single `$(form).find()`, and use `el.type` to see if el is a checkbox, and hence treat it differently.

Reply:
Super clean! :100: